pdal/SpatialReference.cpp

```diff
@@ -237,7 +237,9 @@ void SpatialReference::set(std::string v)
         throw pdal_error(oss.str());
     }
 
-    m_wkt = exportToWkt(&srs);
+    m_epoch = srs.GetCoordinateEpoch();
+
+    m_wkt = exportToWkt(&srs, {"FORMAT=WKT2_2018"});
```

Comment:
Is `WKT2_2018` the default format for WKT export from GDAL right now already? What flavor would you get if you didn't request this?

Reply:
> Is `WKT2_2018` the default format for WKT export from GDAL right now already?

no, it is still WKT1_GDAL

Reply:
Hmm, that might break some stuff that expects WKT1.

Reply:
> Hmm, that might break some stuff that expects WKT1.

shouldn't they use the getWKT1() method then ?

I could potentially limit the scope of that change to situations where I detect a GEOID_MODEL in the WKT, but that would be a bit ugly

Reply:
I think I like this better. That way WKT2+ is opt-in and we don't disrupt anyone who might have been assuming WKT1

Reply:
> I think I like this better. That way WKT2+ is opt-in and we don't disrupt anyone who might have been assuming WKT1

done in the last 3 commits